src/components/CytoscapeGraph/CytoscapeContextMenu.tsx

```diff
@@ -153,10 +153,9 @@ export class CytoscapeContextMenuWrapper extends React.PureComponent<Props> {
         sticky: true,
         interactive: true,
         theme: 'light-border',
-        size: 'large',
         distance: this.tippyDistance(target)
       }
-    ).instances[0];
+    )[0];
 
```

Comment:
Prop Size was removed from here. I am checking where we must to set this so I am learnign about tippy.

On the other hand tippy method now return ``` Instance<TProps> ``` so instances is not required.